### DISCUSSION TITLE
Are we allowed to modify anything after the presentation?

### DISCUSSION THREAD

**Discussion by dariogmori** [Arquisoft/faq#74] [General]
We were allowed to modify some specific things (the e2e) but we also decided that we could just modify some neat picks on the ui because we want to. Is it allowed or should we create our own fork of the project?

**Comment by labra** [comment]
Yes, you can modify anything in the project now and in the future ((the project is also yours). 

However, if you want those changes to be taken into account by the teachers, you should do those changes during this week because next week we will do the final assessment and assign the lab assignment marks. In case you do some change that may be relevant, send an email with copy to the 4 teachers so we can take it into account because otherwise we may not notice it.
